pixi/mnist_gpu.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# detailed logging to stderr
+set -x
+
+echo -e "# Hello CHTC from Job ${1} running on $(hostname)\n"
+echo -e "# GPUs assigned: ${CUDA_VISIBLE_DEVICES}\n"
+
+echo -e "# Installing Pixi"
+curl -fsSL https://pixi.sh/install.sh | bash
+. ~/.bashrc
+
+echo -e "\n# Check to see if the NVIDIA drivers can correctly detect the GPU:\n"
+nvidia-smi
+
+# Executing a Pixi command also installs the environment
+echo -e "\n# Check if PyTorch can detect the GPU:\n"
+curl -sLO https://raw.githubusercontent.com/matthewfeickert/nvidia-gpu-ml-library-test/5d6165b2222dde67c16550f6fb595907ce7b0ce6/torch_detect_GPU.py
+time pixi run python ./torch_detect_GPU.py
```

Comment:
Installing the environment seems to take about 70 seconds in testing on the worker nodes. The install is multiple GBs, so this seems to be a testimony to CHTC's networking.